csrankings-a.csv

```diff
@@ -331,7 +331,7 @@ Ahmad Sharieh,University of Jordan,http://eacademic.ju.edu.jo/sharieh,lXeTzzAAAA
 Ahmad-Reza Sadeghi,TU Darmstadt,https://www.trust.informatik.tu-darmstadt.de/people/ahmad-reza-sadeghi,p66Ie-YAAAAJ
 Ahmed Abdelali,Qatar Computing Research Institute,http://www.qcri.org/our-people/bio?pid=103&amp;par=acc&amp;name=AhmedAbdelali,ib1BYugAAAAJ
 Ahmed Abu-Hajar,Washington State University,https://school.eecs.wsu.edu/people/faculty/ahmed-abu-hajar,NOSCHOLARPAGE
-Ahmed Aleroud,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=AALEROUD,vW6NVIQAAAAJ
+Ahmed F.  Al-Eroud,Augusta University,https://www.augusta.edu/faculty/directory/view.php?id=AALEROUD,vW6NVIQAAAAJ

```

Comment:
There are three aliases listed for this author in DBLP, and all three need entries (FWIW, the extra space after the "F." will cause problems too). See https://dblp.org/pid/85/9730.html.